packages/components/src/components/data-list/data-list-row.tsx

```diff
@@ -47,12 +48,25 @@ const DataListRow = ({
 }: TDataListRow) => {
     const [show_desc, setShowDesc] = React.useState(false);
     const isMounted = useIsMounted();
+    const debouncedHideDetails = useDebounce(() => setShowDesc(false), 5000);
```

Comment:
Why do we need debounce to show desc?

Reply:
When user clicks on position (`setShowDesc(true)`), we are showing text content. After 5 sec state should be set back to `false` in order to hide the phrase. But user also can hide it by clicking on it. So, we want to handle such cases, when user clicks multiple times (like on the video)
https://github.com/binary-com/deriv-app/assets/121025168/da153c24-0b26-4312-815e-6faad5b8a842

Reply:
Alright! Makes sense. Thanks for the explanation 🙏